demo/demo_director.py

```diff
@@ -407,9 +407,12 @@ def register_vehicle_manifest_wrapper(
   be extracted before it is passed to the underlying director.py (in the
```

Comment:
Raw data has to be extracted from the xmlrpc Binary() object before it is passed to director.register_vehicle_manifest()